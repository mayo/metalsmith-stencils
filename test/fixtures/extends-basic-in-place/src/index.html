---
template: template.html
name: First Last
---

{{name}}
