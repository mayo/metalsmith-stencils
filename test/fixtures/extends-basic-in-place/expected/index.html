<html>

<body>

First Last

</body>

</html>
